pulp_ansible/tests/functional/cli/test_collection_install.py

```diff
@@ -77,3 +83,96 @@ def test_install_collection(self):
             subprocess.run(cmd.split())
 
             self.assertTrue(path.exists(directory), "Could not find directory {}".format(directory))
+
+    @skip_if(bool, "AH_token", False)
+    def test_install_collection_with_token_from_automation_hub(self):
```

Comment:
Tested locally and it is working! It will only run on pulp org, as travis does not set variables on PRs